packages/firebase_auth/firebase_auth_platform_interface/lib/src/additional_user_info.dart

```diff
@@ -13,11 +13,11 @@ class AdditionalUserInfo {
       {this.isNewUser, this.profile, this.providerId, this.username});
 
   /// Whether the user account has been recently created.
-  final bool isNewUser;
+  final bool /*!*/ isNewUser;
 
   /// A [Map] containing additional profile information from the identity
   /// provider.
-  final Map<String, dynamic> profile;
+  final Map<String, dynamic> /*!*/ profile;
 
   /// The  federated identity provider ID.
```

Comment:
Just seen it :)
The additional space is useless 
```suggestion
  /// The federated identity provider ID.
```

Reply:
Hm not sure why the formatter didn't pick it up!